perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -118,23 +145,23 @@ def _ResultFilePath(vm):
                         vm.hostname + '.stress_results.txt')
 
 
-def RunTestOnLoader(vm, data_node_ips):
+def RunTestOnLoader(vm, loader_index, keys_per_vm, data_node_ips):
   """Run Cassandra-stress test on loader node.
 
   Args:
     vm: The target vm.
+    loader_index: The index of target vm in loader vms.
+    keys_per_vm: The number of keys per loader vm need to insert.
     data_node_ips: List of IP addresses for all data nodes.
   """
   vm.RobustRemoteCommand(
-      '%s '
-      '--file "%s" --nodes %s '
-      '--replication-factor %s --consistency-level %s '
-      '--num-keys %s -K %s -t %s' % (
-          CASSANDRA_STRESS,
-          _ResultFilePath(vm),
-          ','.join(data_node_ips),
-          REPLICATION_FACTOR, CONSISTENCY_LEVEL, FLAGS.num_keys,
-          RETRIES, FLAGS.num_cassandra_stress_threads))
+      '%s write cl=%s n=%s '
```

Comment:
Nit: as this expression becomes more complex, using `str.format` with named arguments would be nice.